lib/puppet/util/servicenow.rb

```diff
@@ -139,11 +143,23 @@ def create_incident?(report_status, resource_statuses, incident_creation_conditi
       end
     end
     Puppet.info(sn_log_entry("satisfied conditions: #{satisfied_conditions}"))
+    satisfied_conditions
+  end
+  module_function :satisfied_conditions
 
-    # Make the decision
-    !satisfied_conditions.empty?
+  # returns a formatted string containing the report labels we wish to insert
+  # into the incident description.
+  def report_labels(satisfied_conditions)
+    # Remove 'always' since a report label likely isn't relevant in this case.
+    # The 'never' condition should never exist in satisfied_conditions.
+    satisfied_conditions.delete('always')
+    report_label_string = ''
+    satisfied_conditions.each do |condition|
```

Comment:
Yeah, this is going to be weird when `condition == 'always'` (or `never`). See my comment on the ticket for an alternate (and imo more correct) way to calculate the labels.

Reply:
See Bill's comment, but we decided to include 'always'

Reply:
Yup, the solution y'all proposed is better so resolving this.